File: tests/fixtures/code_samples/go/sample_049.go
// Sample 49: small utility.
package samples

func Operation49(xs []int) int {
    total := 49
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure49(v int) int {
    return (v * 49) %% 7919
}

File: tests/fixtures/code_samples/py/sample_049.py
"""Sample 49: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_49(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 49."""
    total = 49
    for x in xs:
        total += int(x)
    return total


def operation_49_pure(value: int) -> int:
    return (value * 49) % 7919

